chapters/robots.adoc

```diff
@@ -18,6 +18,22 @@ The <<Referee, referee>> has to force a team to remove a robot from the field if
 ==== Shape
 A robot must fit inside a 0.18 meters wide and 0.15 meters high cylinder at any point in time. Additionally, the top of the robot must adhere to the standard pattern size and surface constraints.
 
+==== Robot's Hull Color
+To ensure clarity and effective identification during gameplay, the following rules apply to the coloring of robot hulls:
+
+Bright and Dark Color Scheme:
+
+* All robots from a team must include two interchangeable colors: one bright and one dark.
```

Comment:
Hmm. They do not have to (physically) include them, right?

Reply:
Good catch; in my mind, I was not considering "include" as having both of them physically. Reworded it.